webpack.config.js

```diff
@@ -225,6 +225,12 @@ module.exports = [
         },
         performance: getPerformanceHints(5 * 1024)
     }),
+    Object.assign({}, config, {
+        entry: {
+            'close3': './static/close3.js'
+        },
+        performance: getPerformanceHints(30 * 1024)
```

Comment:
Do we need the size limit to be 30K? Or we can put smaller limit?

Reply:
Initially I set it to 5K and the bundle was just a touch over that so I copy-pasta'd the closest one, but you're right, it can be smaller :D